extensions/objects/templates/svg-filename-editor.component.html

```diff
@@ -10,7 +10,33 @@
     <div class="fabric-canvas" ng-attr-id="<[$ctrl.canvasContainerId]>">
       <canvas ng-attr-id="<[$ctrl.canvasID]>" class="fabric-canvas-container"></canvas>
     </div>
-    <div ng-if="$ctrl.isDrawModePieChart()" class="fabric-canvas pie-chart-form">
+    <div ng-if="$ctrl.isDrawModeSVGUpload()" class="fabric-canvas custom-tool-form">
```

Comment:
Just a note -- I think we prefer isDrawModeSvgUpload(), we treat other acronyms similarly. Maybe fix throughout?

Reply:
I have changed SVG to Svg everywhere except for the fabric js functions. Done.